src/main/java/dude/commands/Parser.java

```diff
@@ -0,0 +1,54 @@
+package dude.commands;
+
+import dude.task.ListManager;
+import dude.task.Todo;
+
+import java.util.List;
+
+public class Parser {
+    public static void parseInput(String input) {
+        String[] commands = input.split(" ",2);
```

Comment:
coding standards: there should be space after comma
`String[] commands = input.split(" ", 2);`